### ISSUE TITLE
App crashing with pino 5.x and prettyPrint

### ISSUE THREAD

**Issue by devourment77** [pinojs/hapi-pino#65]
Hello,

I have a lot of large JSON objects being logged to stdout and I have been getting the following inconsistently:

```
Error: EAGAIN: resource temporarily unavailable, write
Emitted 'error' event at:
    at SonicBoom.release
    at FSReqWrap.wrapper [as oncomplete] (fs.js:500:5)
[nodemon] app crashed - waiting for file changes before starting...
```

Assuming it is related to pino 5.x update the sonic boom streaming switch?

Here is my plugin config:

```
export default {
  plugin: Pino, // this is hapi-pino
  options: {
    name: 'api',
    prettyPrint: config.env !== 'production',
    level: config.logger.level,
    // logPayload: config.env !== 'production',
    logRouteTags: true,
    mergeHapiLogData: false,
    ignorePaths: ['/api/health'],
  },
};
```

**Comment by mcollina** [comment]
Which operating system are you in?

**Comment by mcollina** [comment]
Does it happen in dev or in production? pretty-print or not? Are you piping the output or it just goes to the TTY.

**Comment by mcollina** [comment]
Can you also check if it happens without nodemon?

**Comment by devourment77** [comment]
on it! will keep you posted.

**Comment by mcollina** [comment]
Can you please verify that https://github.com/mcollina/sonic-boom/pull/12 fixes it?

**Comment by devourment77** [comment]
**nodemon**
Happened w/o nodemon.

**prettyPrint**
I only saw it happen with pretty print enabled, was not able to replicate with pretty print off. (manually testing, didn't write a script or anything so it could still be possible and I just didn't hit it).

**sonic-boom #12**
Copy pasta'd the file to node_modules locally and so far so good.  Will keep it in for the rest of the day to see if I hit failure again.

**Comment by mcollina** [comment]
@devourment77 what's the status? Is this working fine?

**Comment by devourment77** [comment]
So far so good (using https://github.com/mcollina/sonic-boom/pull/12 in node_modules locally).

**Comment by mcollina** [comment]
I've released the fix as sonic-boom@0.6.1.
